tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1314,10 +1319,7 @@ Duration <DURATION>
 "
 `;
 
-exports[`panic.w 2`] = `
-"Compiler bug when type-checking ../../../examples/tests/invalid/panic.w:6:1 | panicked at 'User invoked panic', libs/wingc/src/type_check.rs:LINE:COL
-Please report this bug at https://docs.winglang.io/contributors/bugs"
-`;
+exports[`panic.w 2`] = `"Panicked, backtrace not captured: Unsupported"`;
```

Comment:
Is this a regression?

Reply:
No, notice this is the stderr output. Now panics go out as normal diags which in the context of invalid tests are printed to stdout so this output now moved to stdout above (with a slight change in phrasing).
I added an automatic stack trace to stderr in case of panics for the benefit of reporting, but in practice these aren't supported in WASM (unless I'm missing something). So all we get in stderr is a notice stack traces aren't supported, if they ever will be, running the compiler with `RUST_BACKTRACE=1` should produce the trace here (it does when using native code).